__tests__/app.test.js

```diff
@@ -446,4 +511,47 @@ describe('Error-handling Sad paths', () => {
             });
         });
     });
+    describe('Invalid query values', () => {
+        describe('GET /api/articles', () => {
+            describe('topic', () => {
+                it('should return a 404 not found if queried with a non-existent topic value', () => {
+                    return request(app)
+                        .get("/api/articles?topic=notATopic")
+                        .expect(404)
+                        .then(({ body }) => {
+                            expect(body).toStrictEqual({

```

Comment:
_Technically_ any topic could be a valid topic, so although there are no articles that match it might be nice to provide a slightly more informative error message eg `no articles could be found for <topic>`